cooperator/models/account_move.py

```diff
@@ -66,11 +66,13 @@ def get_mail_template_certificate(self):
             mail_template = "cooperator.email_template_certificat"
         return self.env.ref(mail_template)
 
-    def get_sequence_register(self):
-        return self.env.ref("cooperator.sequence_subscription", False)
+    def get_sequence_number(self):
```

Comment:
shouldn’t it be `get_number_sequence()` (or `get_cooperator_number_sequence()`), since it returns a sequence and not a number?